### ISSUE TITLE
Does fastlane for upload to playstore function releases app to production track as default

### ISSUE THREAD

**Issue by vickydhas** [fastlane/fastlane#22165]
### New Issue Checklist

- [X] Updated fastlane to the latest version
- [X] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [X] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [X] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
Does fastlane for upload to playstore function releases app to production track as default, as per the [documentation](https://docs.fastlane.tools/actions/upload_to_play_store/) guideline if no parameter for **track**, **release_status** are passed its uploaded to production track.

<img width="1072" alt="image" src="https://github.com/user-attachments/assets/bb43adfe-030f-4326-b45e-41768f3146e3">

### Question

1. Does fastlane library starts uploading to production track directly 
2. As the release_status parameter is not passed will the library release the app straight to production using the APIs inside.
3. If yes how much % of rollout will be made live to users.

##### Command executed
`fastlane lane upload_app_to_playstore`

##### Complete output when running fastlane, including the stack trace and command used
_When no parameters passed to the **track** and **status**:_
<img width="407" alt="image" src="https://github.com/user-attachments/assets/d6d9ca9e-7023-4ba0-b76f-2f181c5b2d20">

_Output as below:_ 
![image](https://github.com/user-attachments/assets/289de911-f797-4a0f-802e-3a7dacbb52dc)

<details>
  <pre> 

1. Few enquiries as the managed publishing is on will the release to internal track  also publish the binary to production track as its available on that track
2. Is this just upload only or also make live to customers as this is not send with draft parameter in release status
3. Will the binary be available to customers at 100%
</pre>
</details>

**Comment by esteban78s** [comment]
Yes, by default, fastlane uploads apps to the production track if no track or release status is specified. To avoid accidental releases, always specify the desired track and release status, including the rollout percentage to control the release pace.

**Comment by iBotPeaches** [comment]
1. By default yes.
2. `completed` is default
3. Unless you explicitly specify it between 0/1 - whatever is configured in your account.

Closing as not a bug, but support.
